src/Model/MailChimpList.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+namespace Dynamic\FoxyStripeMailChimp\Model;
+
+use SilverStripe\ORM\DataObject;
+
+/**
+ * Class MailChimpList
+ */
+class MailChimpList extends DataObject
+{
+    private static $db = array(
+        'Title' => 'Varchar(255)',
+        'MCID' => 'Varchar(255)',
+    );
+
+    private static $has_many = array(
+        'MailingSegments' => MailChimpSegment::class,
+    );
+
+    private static $indexes = array(
+        'MCID' => array(
+            'type' => 'unique',
+            //'value' => 'MCID',
```

Comment:
@mak001 I had issues with the index value so I commented out, could you take a look?